src/http.c

```diff
@@ -45,6 +45,8 @@ static void url_encode(char *dest, const char *str)
 		if (isalnum(*str) || *str == '-' || *str == '_'
 		    || *str == '.' || *str == '~')
 			*dest++ = *str;
+//                else if (*str == ' ')
+//                        *dest++ = '+';
```

Comment:
I had a quick look. We can omit these two lines that are commented out anyway (actually we have removed them last week on the master branch). 
I haven't found the time to test yet, but from quickly looking over the code changes that's the only thing that I can see.

Reply:
Ok - I've cleaned that up.

Reply:
I did some testing in a Virtualbox... 

- Fedora 28 and 29 - OK
- SuSE Tumbleweed - OK
- Suse Leap 15.0 and 15.1 - OK, but needs a newer version of libp11!
- Ubuntu 16.04 LTS - OK
- MacOS Mojave - NOK, it doesn't find the pkcs engine-by-id. Will have a deeper look on it.